release/sapradhan/sapradhan.new-newa.nepal_bhasa/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+(cc) 2020 Ananda K Maharjan, Santosh Pradhan

```

Comment:
```suggestion
(c) 2020 Ananda K Maharjan, Santosh Pradhan
```

Reply:
can it be left to creative commons license ?

Reply:
We chose the MIT license in the lexical-models repo to follow the Keyman keyboards repo. 

See the bottom of this page for more info (Assigning an open source license):
https://help.keyman.com/developer/keyboards/advanced/step-0

Reply:
changed as requested